app/javascript/components/Header/BasicHeader.tsx

```diff
@@ -7,7 +7,7 @@ import './BasicHeader.css';
 
 const logoImg = 'https://search.gov/assets/gsa-logo-893b811a49f74b06b2bddbd1cde232d2922349c8c8c6aad1d88594f3e8fe42bd097e980c57c5e28eff4d3a9256adb4fcd88bf73a5112833b2efe2e56791aad9d.svg';
 
-export const BasicHeader = (props: HeaderProps) => {
+export const BasicHeader = ({ title, toggleMobileNav, mobileNavOpen }: HeaderProps) => {
```

Comment:
I've added prop destructuring to all React components in this PR. This means we can access props as `mobileNavOpen` instead of `props.mobileNavOpen`.